plugins/callback/mail.py

```diff
@@ -94,34 +95,36 @@ def set_options(self, task_keys=None, var_options=None, direct=None):
         self.cc = self.get_option('cc')
         self.bcc = self.get_option('bcc')
 
+    @staticmethod
+    def encode_addresses(addresses):
+        addresses_list = addresses.split(',')
+        addresses_list = map(lambda x: email.utils.formataddr(email.utils.parseaddr(x), charset='utf-8'), addresses_list)
```

Comment:
Please do not use `map` and `lambda`, but list comprehensions:
```suggestion
        addresses_list = [email.utils.formataddr(email.utils.parseaddr(x), charset='utf-8') for x in addresses_list]
```
That's much easier to read.